docs/reference/openapi.yaml

```diff
@@ -1,481 +1,289 @@
-      type: object
-    Task:
-      additionalProperties: false
-      description: Task that will run a plan
-      properties:
-        name:
-          description: Name of plan to run
-          title: Name
-          type: string
-        params:
-          description: Values for parameters to plan, if any
-          title: Params
-          type: object
-      required:
-      - name
-      title: Task
-      type: object
-    TaskResponse:
-      additionalProperties: false
-      description: Acknowledgement that a task has started, includes its ID
-      properties:
-        task_id:
-          description: Unique identifier for the task
-          title: Task Id
-          type: string
-      required:
-      - task_id
-      title: TaskResponse
-      type: object
-    TasksListResponse:
-      additionalProperties: false
-      description: Diagnostic information on the tasks
-      properties:
-        tasks:
-          description: List of tasks
-          items:
-            $ref: '#/components/schemas/TrackableTask'
-          title: Tasks
-          type: array
-      required:
-      - tasks
-      title: TasksListResponse
-      type: object
-    TrackableTask:
-      additionalProperties: false
-      description: A representation of a task that the worker recognizes
-      properties:
-        errors:
-          items:
-            type: string
-          title: Errors
-          type: array
-        is_complete:
-          default: false
-          title: Is Complete
-          type: boolean
-        is_pending:
-          default: true
-          title: Is Pending
-          type: boolean
-        task:
-          title: Task
-        task_id:
-          title: Task Id
-          type: string
-      required:
-      - task_id
-      - task
-      title: TrackableTask
-      type: object
-    ValidationError:
-      properties:
-        loc:
-          items:
-            anyOf:
-            - type: string
-            - type: integer
-          title: Location
-          type: array
-        msg:
-          title: Message
-          type: string
-        type:
-          title: Error Type
-          type: string
-      required:
-      - loc
-      - msg
-      - type
-      title: ValidationError
-      type: object
-    WorkerState:
-      description: The state of the Worker.
-      enum:
-      - IDLE
-      - RUNNING
-      - PAUSING
-      - PAUSED
-      - HALTING
-      - STOPPING
-      - ABORTING
-      - SUSPENDING
-      - PANICKED
-      - UNKNOWN
-      title: WorkerState
-      type: string
-    WorkerTask:
-      additionalProperties: false
-      description: Worker's active task ID, can be None
-      properties:
-        task_id:
-          anyOf:
-          - type: string
-          - type: 'null'
-          description: The ID of the current task, None if the worker is idle
-          title: Task Id
-      required:
-      - task_id
-      title: WorkerTask
-      type: object
+openapi: 3.1.0
```

Comment:
These changes seem unrelated, if possible I'd like the openapi.yaml to remain unchanged, or at least reduce the size of the changes to this file. Do we know why it needs to be regenerated? If it isn't are there changes?